### ISSUE TITLE
Bug: Element.Faces, Element.Geometry, Element.Solids show output and warning for PolyCurves may be branching

### ISSUE THREAD

**Issue by catafest** [DynamoDS/DynamoRevit#2628]
_If this issue is **not** a bug report or improvement request, please check the [Dynamo forum](https://forum.dynamobim.com/), and start a thread there to discuss your issue._
I cannot find a way to fix this with A.P.I. python or to parse the model element in order to have a clean running process.
Element model's can come with output and warnings for Element.Faces, Element.Geometry, Element.Solids.
I check also the  [Dynamo forum](https://forum.dynamobim.com/) without a clear solution.
## Dynamo version
2.3.0.5885
## Revit version
2.3.0.8352
## Operating system
Windows 10
## What did you do? 
I use Select Model Element to select an model element and use it with _Element.Faces, Element.Geometry, Element.Solids_.
## What did you expect to see?
1. No Warning;
2. output of each node: Element.Faces, Element.Geometry, Element.Solids;
3. to filter all elements from Element Model and don't send warnings;
4. A.P.I implementation to take and elements use these errors of this simple process and send valid output;
## What did you see instead?
Warning: One or more geometries have failed to convert due to this error: _...PolyCurves may be branching... or something else_ 
I have output for each node depending each Element Model selected:  _Arc , Lines and Solid on Element.Geometry ..._

**Comment by ZiyunShang** [comment]
Hi @catafest , could you provide your Revit model and Dynamo scripts, so I can reproduce on my side?

**Comment by catafest** [comment]
no, sorry , the dyn is on team work and I not allow to do that ... also I see similar questions on web ... you can search for more info.

**Comment by johnpierson** [comment]
Closing this out. A reproduction case was requested and never arrived, so there is nothing here we can investigate.

If you can still reproduce this on a current Dynamo and Revit, please open a fresh issue with the .dyn and a sample .rvt attached and we will dig in.
